spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require "minitest/spec"
 require "minitest/hooks/default"
 require "minitest/reporters"
+require "byebug"
```

Comment:
I'm constantly importing and using `byebug` during debugging, so I included it in our `spec_helper`. It seems like a useful place to have it included, however, if there's strong opposition, I can purge this.